src/ring_theory/polynomial/hermite/basic.lean

```diff
@@ -127,4 +129,59 @@ end
 
 end coeff
 
+section coeff_explicit
+
+open_locale nat
+
+/- Because of `coeff_hermite_of_odd_add`, every nonzero coefficient is described as follows.  -/
```

Comment:
Did you intend for this to be a doc comment that appears in hover text and on the web?
```suggestion
/-- Because of `coeff_hermite_of_odd_add`, every nonzero coefficient is described as follows.  -/
```

Reply:
Ah, I wasn't sure what type of comment was appropriate here, actually. Some comment is necessary to explain why the formula is for `(hermite (2 * n + k)).coeff k` rather than `(hermite n).coeff k`.